integration/core/test_api.py

```diff
@@ -11,3 +12,34 @@ def client():
 def test_catalog_list(client):
     templates = client.list_template()
     assert len(templates) > 0
+
+
+def test_templatebase_eq_filter(client):
+    url = 'http://localhost:8088/v1-catalog/templates?templateBase_eq=invalid'
```

Comment:
@ibuildthecloud I cannot add that test right now until the rancher-catalog supports the additional folder. Until then there is nothing other than the regular templates to list.
I will add this test when the catalog starts having kubernetes-templates. does that sound fine?